source/connect/connection-options/connection-pools.txt

```diff
@@ -31,21 +31,62 @@ are created by {+driver-short+}.
 
 .. _node-faq-connection-pool:
 
-Connection Pool Overview
--------------------------
+Configure Connection Pools
+--------------------------
 
 Every ``MongoClient`` instance has a built-in connection pool for each server
 in your MongoDB topology. Connection pools open sockets on demand to
 support concurrent requests to MongoDB in your application.
 
-The maximum size of each connection pool is set by the ``maxPoolSize`` option, which
-defaults to ``100``. If the number of in-use connections to a server reaches
-the value of ``maxPoolSize``, the next request to that server will wait
-until a connection becomes available.
+You can specify the following connection pool settings in your ``MongoClient``
+instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Setting
+     - Description
+
+   * - ``maxPoolSize``
+     - | The maximum number of concurrent connections that the pool maintains.
+         If the number of in-use connections to a server reaches the specified
+         value, the next request to that server waits until a connection becomes
+         available.
+       |
+       | **Default**: ``100``
+
+   * - ``maxConnecting``
+     - | The maximum number of connections that each pool can establish
+         concurrently.
+
+   * - ``minPoolSize``
+     - | The minimum number of concurrent connections that the pool maintains.
+       |
+       | **Default**: ``0``
+
+   * - ``maxIdleTimeMS``
+     - | The maximum number of milliseconds that a connection can remain idle in
+         the pool.
+       |
+       | **Default**: ``0`` (no limit)
+
+   * - ``waitQueueTimeoutMS``
+     - | The maximum number of milliseconds that a request can wait for a socket
+         to become available.
+       |
+       | **Default**: ``0`` (no limit)
+
+
+.. _node-connection-pool-max-pool-size:
+
+maxPoolSize
+~~~~~~~~~~~
 
 In addition to the sockets needed to support your application's requests,
 each ``MongoClient`` instance opens two more sockets per server
 in your MongoDB topology for monitoring the server's state.
```

Comment:
Not sure how precise we want to be with the language here, but the number of connections opened by driver depends on how the driver is monitoring.  In most cases, it is 2 per server, but that isn't true if:

1. the user is using a load balancer (no monitoring is done if there is a load balancer configured)
2. the user is using a pre-4.4 server (then we only use 1 connection per server)
3. the user is operating in a FAAS environment, like AWS Lambda (then there will only be one connection per server)

Maybe we could say "up to two connection per server"?